README.md

```diff
@@ -39,6 +43,7 @@ The buildpack will do the following:
 | `$BP_SPRING_CLOUD_BINDINGS_DISABLED`  | Whether to contribute Spring Cloud Bindings support to the image at build time.  Defaults to false.                                                                                                                                                                     |
 | `$BPL_SPRING_CLOUD_BINDINGS_DISABLED` | Whether to auto-configure Spring Boot environment properties from bindings at runtime. This requires Spring Cloud Bindings to have been installed at build time or it will do nothing. Defaults to false.                                                               |
 | `$BPL_SPRING_CLOUD_BINDINGS_ENABLED`  | Deprecated in favour of `$BPL_SPRING_CLOUD_BINDINGS_DISABLED`. Whether to auto-configure Spring Boot environment properties from bindings at runtime. This requires Spring Cloud Bindings to have been installed at build time or it will do nothing. Defaults to true. |
+| `$BP_SPRING_CLOUD_BINDINGS_VERSION`  | Explicit version of Spring Cloud Bindings library to install.  Defaults to `1`.   
```

Comment:
Do you think a mention of the "default" version is worthwhile here (and above)? It kind of seems confusing to me. The real "default" version is what's in `MANIFEST.MF`, right? By that I mean, that file will always exist for a Spring app, right? So it should never use the default in the buildpack, even though we need to set one.

Reply:
So I removed it from the variable but not from the order sequence above. I don't feel too strongly about it but technically if there is an error parsing the Boot value, the default value is the fallback that is used. 
I originally had this case erroring and failing the build, but I thought it was better to fall back to installing a version and proceeding. If you think I should error instead let me know.

If not, I think the mention of the default belongs in the order list because it describes what the code is doing (albeit in the "it'll never happen" scenario 🥴 )?